src/app/dataset/dataset.component.ts

```diff
@@ -69,6 +75,21 @@ export class DatasetComponent implements OnInit, OnDestroy {
   reportMsg?: string;
   reportLoading: boolean;
 
+  @ViewChild(WorkflowComponent) workflowFormRef: WorkflowComponent;
+
+  @ViewChild(WorkflowHeaderComponent) workflowHeaderRef: WorkflowHeaderComponent;
+
+  formInitialised(workflowForm: FormGroup): void {
+    if (this.workflowHeaderRef) {
+      this.workflowHeaderRef.setWorkflowForm(workflowForm);
+    } else {
+      const that = this;
+      setTimeout(() => {
+        that.formInitialised(workflowForm);
```

Comment:
No - this loses context within a setTimeout

Reply:
Try it, you are using an arrow function and then the this will refer to the originating context.
I have tried it and as far as I can see, it works without any trouble.

Reply:
OK - using this instead of that!